### PR TITLE
Query and update animal table

### PR DESCRIPTION
**Pull request by ifeanyi305**
Hi reviewer, for this branch I updated and query the animal table check below to see screenshots of queries used for the table:

## Add a new row

first transaction

## Update table rows and column

first transaction2

first transaction3

first transaction4

## Delete and rollback columns

delete transaction

## Queries 

COUNT_ANIMAL

count-escape

Screenshot 2023-01-18 at 10 32 54 AM

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -1,5 +1,15 @@
...
-INSERT INTO animals (name) VALUES ('Luna');
-INSERT INTO animals (name) VALUES ('Daisy');
-INSERT INTO animals (name) VALUES ('Charlie');
+INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg) 
+VALUES 
+(1, 'Agumon', '2020-02-03', 0, 'TRUE', 10.23),
+(2, 'Gabumon', '2018-11-15', 2, 'TRUE', 8),
+(3, 'Pikachu', '2021-01-07', 1, 'FALSE', 15.04),
+(4, 'Devimon', '2017-05-12', 5, 'TRUE', 11),
+(5, 'Charmander', '2020-02-08', 0, 'FALSE', 11),
```
Comment (BenMukebo) [inline data.sql]: - [ ] Great job so far, but you need to pay more attention to the project requirements. The first data must have a negative weight_kg value and not a positive value. Please fix this 👍🏻Your code might look like this:

```sql
    INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
    VALUES ('Charmander', '2020/2/8', '0', 'false', '-11.0');
```

File: data.sql [hunk 2]
```diff
@@ -1,5 +1,15 @@
...
+(6, 'Plantmon', '2021-11-15', 2, 'TRUE', -5.7),
+(7, 'Squirtle', '1993-04-02', 3, 'FALSE', -12.13),
+(8, 'Angemon', '2005-06-12', 1, 'TRUE', -45),
+(9, 'Boarmon', '2005-06-07', 7, 'TRUE', 20),
```
Comment (BenMukebo) [inline data.sql]: - [ ] The same here,  The weight_kg of the fifth data has a decimal value. Please add this 👍🏻Your code might look like this:

```sql

    INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
    VALUES ('Boarmon', '2005/06/07', '7', 'true', '20.4');

File: queries.sql [hunk 3]
```diff
@@ -1,3 +1,96 @@
...
+
+BEGIN;
+/* Delete a transaction */
+DELETE FROM animals;
+/* Return to previuse state of transation */
+ROLLBACK;
+SELECT * FROM animals;
+
+DELETE FROM animals WHERE date_of_birth > '2022-01-01';
+
```
Comment (BenMukebo) [inline queries.sql]: - [ ] Make sure to  Start the transaction before you `DELETE` all animals born after Jan 1st, 2022 . Eg:

```sql
    BEGIN;

    DELETE FROM animals WHERE  date_of_birth > '2022/01/01' ;
```

File: queries.sql [hunk 4]
```diff
@@ -1,3 +1,96 @@
...
+/* Start transaction */
+BEGIN;
+/* COUNT NUMBER OF ANIMALS */
+SELECT COUNT(*) FROM animals;
+
+SELECT COUNT(*) FROM animals WHERE escape_attempts = 0;
+
+SELECT AVG(weight_kg) FROM animals;
+
+SELECT neutered, SUM(escape_attempts) FROM animals GROUP BY neutered;
```
Comment (BenMukebo) [inline queries.sql:90]: - [OPTIONAL]  Or you could also use MAX =>

  ```sql
    SELECT neutered, MAX(escape_attempts) FROM animals
    GROUP BY neutered;
  ```

**Review by BenMukebo — CHANGES_REQUESTED** [review]
### Hi @ifeanyi305,

You did a great job so far 👏🏼 👏🏼
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

### To highlight 💪🏻
- No linter errors. ✔️ 
- Gitflow was correctly used.  ✔️   
 👍  Good job!


## STATUS: CHANGES REQUESTED  ♻️ 
_Please Check the comments under the review and make all required changes_

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏 💻  
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me @BenMukebo when you comment so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by andersonlebon — APPROVED** [review]
## Hi @ifeanyi305 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

nodding-yes

## Optional suggestions
Screenshot 2023-01-18 at 15 45 38

- [ ] Kindly remember to customize  the  microverse`README` file template with your own information 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question `@andersonlebon` so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**